scheduler/scheduler.go

```diff
@@ -250,6 +276,7 @@ func BuildDependencyGraph(c client.Interface, sel labels.Selector) (DependencyGr
 			depGraph[child].Requires, depGraph[parent])
 
 		depGraph[child].Meta[parent] = d.Meta
+		depGraph[parent].Meta[parent] = d.Meta
```

Comment:
Hmmm.... shouldn't there be `.Meta[child]`?
Looks like this is overwriting something.